frontend/cypress/integration/common/wizard_request_routing.ts

```diff
@@ -158,4 +168,23 @@ And('user sees the {string} regex in the editor', (regexContent) => {
     cy.get('.ace_content')
         .invoke('text')
         .should('match', re);
+});
+
+And('user clicks on Show Advanced Options', () => {
+    cy.get('.pf-c-expandable-section__toggle-text')
```

Comment:
Sorry for my delay in this review.
I was also checking if this selector could be changed for some custom data-test attribute, but it seems that the component is wrapped in a way that there is no obvious way to add data- attributes or reuse an existing aria- attribute.

Reply:
I was wondering if "contentId" could make the trick here, to avoid to query just the style:
![image](https://user-images.githubusercontent.com/1662329/169825832-6da48e25-4151-4bce-8b77-8e233508b400.png)

But not a blocker.

Reply:
+1, thanks for the hint and review